docker-entrypoint.d/00-update-uid

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# current www-data UID
+CURRENT_UID=`id -u www-data`
+WANTED_UID=${RUN_AS_UID:-33}
+
+# check if UID need to be updated
+if [ $CURRENT_UID -ne $WANTED_UID ]; then
```

Comment:
`CURRENT_UID` and `WANTED_UID` should not be empty because :
* `id` command always return numeric identifier
* `${RUN_AS_UID:-33}` use 33 as default value if `RUN_AS_UID` is not set

Can you explain steps to reproduce your issue, maybe your `www-data` default uid was not set to 33.